Pokemon Unity/Assets/Scripts2/Pokemon/Pokemon.cs

```diff
@@ -1481,15 +1522,155 @@ public void clearAllRibbons()
         #endregion
 
         #region Items
-        #endregion
+		public Items SwapItem(Items item)
+		{
+			Items old = Item;
+			Item = item;
+			return old;
+		}
+
+		#region Mail
```

Comment:
Ugh... lines 1572-1640 wasnt supposed to be in here. It was an accident, and i thought i deleted it... :/ I'll have to get rid of it later... for now, it's not doing any harm.